docs/en/drivers/windows.md

```diff
@@ -0,0 +1,136 @@
+## The Windows Driver
+
+Appium has the ability to automate Windows PC Desktop apps. This driver relies
```

Comment:
This driver also automates Windows mobile apps, right?

Reply:
no, it does not :-(